indico/modules/events/logs/client/js/actions.js

```diff
@@ -33,6 +34,10 @@ export function setDetailedView(entryIndex) {
   return {type: SET_DETAILED_VIEW, currentViewIndex: entryIndex};
 }
 
+export function setDataQuery(dataQuery) {
+  return {type: SET_DATA_QUERY, dataQuery};
```

Comment:
"Data query" seems like a very broad name to me. I know you probably want to avoid "filter" since that's already used for the regular filters in the event log. Maybe "metadataQuery" would make it a bit more intuitive?